<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - Class com.commercetools.pspadapter.payone.PaymentHandlerTest</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>Class com.commercetools.pspadapter.payone.PaymentHandlerTest</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/com.commercetools.pspadapter.payone.html">com.commercetools.pspadapter.payone</a> &gt; PaymentHandlerTest</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">9</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">0</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">2.384s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox success" id="successRate">
<div class="percent">100%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Tests</a>
</li>
<li>
<a href="#tab1">Standard output</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="success">returnsStatusCodeAccepted202InCaseOfOngoingConcurrentPaymentHandling</td>
<td>2.010s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeBadRequest400InCaseOfPaymentWithOtherPaymentInterface</td>
<td>0.001s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromCommercetoolsQuery</td>
<td>0.004s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentDispatcher</td>
<td>0.010s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeInternalServerError500InCaseOfUnexpectedExceptionFromPaymentWithCartLike</td>
<td>0.012s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeNotFound404InCaseOfCartLikeMissing</td>
<td>0.007s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeNotFound404InCaseOfUnknownPayment</td>
<td>0.028s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeOk200InCaseOfSuccessfulConcurrentPaymentHandling</td>
<td>0.312s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">returnsStatusCodeOk200InCaseOfSuccessfulPaymentHandling</td>
<td>0s</td>
<td class="success">passed</td>
</tr>
</table>
</div>
<div id="tab1" class="tab">
<h2>Standard output</h2>
<span class="code">
<pre>10:44:16.143 ?[1;31m[ERROR]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Error on processing payment with ID [2044803381] as payment or cart could not be found
io.sphere.sdk.client.NotFoundException: http response: &lt;unknown&gt;
SDK: 1.29.0
project: &lt;unknown&gt;
Java: 1.8.0_151
cwd: /home/travis/build/commercetools/commercetools-payone-integration/service
sphere request: &lt;unknown&gt;
additional notes: []
Javadoc: http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/NotFoundException.html

	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
10:44:16.155 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1509544928]. Retry 1 of 20
10:44:16.256 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1509544928]. Retry 2 of 20
10:44:16.356 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1509544928]. Retry 3 of 20
10:44:16.461 ?[1;31m[ERROR]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Completion exception error: com.commercetools.pspadapter.payone.PaymentHandlerTest$1: 1916521293
Cause: null
10:44:16.463 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Wrong payment interface name: expected [TestIntegrationServicePaymentMethodName], found [other than TestIntegrationServicePaymentMethodName]
10:44:16.465 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 1 of 20
10:44:16.565 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 2 of 20
10:44:16.666 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 3 of 20
10:44:16.766 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 4 of 20
10:44:16.867 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 5 of 20
10:44:16.967 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 6 of 20
10:44:17.067 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 7 of 20
10:44:17.168 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 8 of 20
10:44:17.268 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 9 of 20
10:44:17.369 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 10 of 20
10:44:17.469 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 11 of 20
10:44:17.570 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 12 of 20
10:44:17.670 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 13 of 20
10:44:17.770 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 14 of 20
10:44:17.871 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 15 of 20
10:44:17.972 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 16 of 20
10:44:18.072 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 17 of 20
10:44:18.173 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 18 of 20
10:44:18.273 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 19 of 20
10:44:18.374 ?[31m[WARN ]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Exception on dispatchPayment with id [1475591794]. Retry 20 of 20
10:44:18.474 ?[1;31m[ERROR]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - The payment [1475591794] couldn't be processed after 20 retries
10:44:18.486 ?[1;31m[ERROR]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Error in response: 
java.lang.RuntimeException: 881410983
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
10:44:18.490 ?[1;31m[ERROR]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Error on processing payment with ID [1286190531] as payment or cart could not be found
com.commercetools.pspadapter.payone.domain.ctp.exceptions.NoCartLikeFoundException: No Order or Cart found for the payment
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
10:44:18.509 ?[1;31m[ERROR]?[0;39m ?[36mc.c.p.p.PaymentHandler[tenant testTenantName]?[0;39m [Test worker] - Error in response: 
</pre>
</span>
</div>
</div>
<div id="footer">
<p>
<div>
<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
</label>
</div>Generated by 
<a href="http://www.gradle.org">Gradle 4.5.1</a> at Nov 28, 2019 10:48:45 AM</p>
</div>
</div>
</body>
</html>
